stdlib/stdlib.mli

```diff
@@ -1080,7 +1080,11 @@ val open_in_gen : open_flag list -> int -> string -> in_channel
    as described above. The extra arguments
    [mode] and [perm] specify the opening mode and file permissions.
    {!Stdlib.open_in} and {!Stdlib.open_in_bin} are special
-   cases of this function. *)
+   cases of this function. 
```

Comment:
```suggestion
   cases of this function.
```
For there to be a newline, ocamldoc needs a blank line following (assuming that was what you intended)?